docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "docs.bruno",
+  "type": "collection"
+}
```

Comment:
It's a good practice to add a newline at the end of the file to avoid any potential issues with file processing tools that expect or require it. [medium]